Splunk py raise timezone exception (#27316)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_0_20.md

#### Integrations

##### SplunkPy

> - Fixed an issue where the returned error message when enabling mirroring without specifying a timezone was not informative.

Reviewer: ```suggestion
- Fixed an issue where the it was possible to configure mirroring without specifying a timezone.
```
